include/librealsense2/h/rs_frame.h

```diff
@@ -62,9 +62,23 @@ typedef enum rs2_frame_metadata_value
     RS2_FRAME_METADATA_FRAME_LED_POWER                      , /**< Led power value 0-360. */
     RS2_FRAME_METADATA_RAW_FRAME_SIZE                       , /**< The number of transmitted payload bytes, not including metadata */
     RS2_FRAME_METADATA_GPIO_INPUT_DATA                      , /**< GPIO input data */
-    RS2_FRAME_METADATA_SEQUENCE_NAME                         , /**< sub-preset id */
-    RS2_FRAME_METADATA_SEQUENCE_ID                , /**< sub-preset sequence id */
-    RS2_FRAME_METADATA_SEQUENCE_SIZE              , /**< sub-preset sequence size */
+    RS2_FRAME_METADATA_SEQUENCE_NAME                        , /**< sub-preset id */
+    RS2_FRAME_METADATA_SEQUENCE_ID                          , /**< sub-preset sequence id */
+    RS2_FRAME_METADATA_SEQUENCE_SIZE                        , /**< sub-preset sequence size */
+    RS2_FRAME_METADATA_SAFETY_DEPTH_FRAME_COUNTER,
+    RS2_FRAME_METADATA_SAFETY_LEVEL1,
+    RS2_FRAME_METADATA_SAFETY_LEVEL1_ORIGIN,
+    RS2_FRAME_METADATA_SAFETY_LEVEL2,
+    RS2_FRAME_METADATA_SAFETY_LEVEL2_ORIGIN,
+    RS2_FRAME_METADATA_SAFETY_LEVEL1_VERDICT,
+    RS2_FRAME_METADATA_SAFETY_LEVEL2_VERDICT,
+    RS2_FRAME_METADATA_SAFETY_HUMAN_VOTE_RESULT,
+    RS2_FRAME_METADATA_SAFETY_HARA_EVENTS,
+    RS2_FRAME_METADATA_SAFETY_SOC_FUSA_EVENTS,
+    RS2_FRAME_METADATA_SAFETY_SOC_FUSA_ACTION,
+    RS2_FRAME_METADATA_SAFETY_FUSA_EVENT,
+    RS2_FRAME_METADATA_SAFETY_FUSA_ACTION,
+    RS2_FRAME_METADATA_SAFETY_CRC32,
```

Comment:
As discussed 
Currently CRC32 is considered as a metadata field and not being checked to verify the message content.
We need to find a way to use it to verify the safety stream metadata content, (and maybe fail the frame?)

Reply:
TBD